recipe/meta.yaml

```diff
@@ -56,9 +62,11 @@ about:
   summary: Helper tool for recipes on aggregate.
   description: |
     Renders local recipes, provides build orders, find outdated recipes.
-  doc_url: https://github.com/anaconda/conda-recipe-manager
+  doc_url: https://conda-incubator.github.io/conda-recipe-manager/index.html
   dev_url: https://github.com/anaconda/conda-recipe-manager
 
 extra:
   recipe-maintainers:
     - conda-incubator/builds-tools
```

Comment:
Is that even a possible entry?

Reply:
I'm not sure I follow, we are hosting the docs on GitHub: https://conda-incubator.github.io/conda-recipe-manager/index.html

The static pages get auto-updated by a GHA. That action was a template from GitHub.